hw/ip/gpio/doc/checklist.md

```diff
@@ -25,15 +25,15 @@ Code Quality  | [LINT_SETUP][]        | Done        |
 [GPIO CSR]: ../data/gpio.hjson
 
 
-[SPEC_COMPLETE]:      {{<relref "/doc/project/checklist.md#spec-complete" >}}
-[CSR_DEFINED]:        {{<relref "/doc/project/checklist.md#csr-defined" >}}
-[CLKRST_CONNECTED]:   {{<relref "/doc/project/checklist.md#clkrst-connected" >}}
-[IP_TOP]:             {{<relref "/doc/project/checklist.md#ip-top" >}}
-[IP_INSTANTIABLE]:    {{<relref "/doc/project/checklist.md#ip-instantiable" >}}
-[MEM_INSTANCED_80]:   {{<relref "/doc/project/checklist.md#mem-instanced-80" >}}
-[FUNC_IMPLEMENTED]:   {{<relref "/doc/project/checklist.md#func-implemented" >}}
-[ASSERT_KNOWN_ADDED]: {{<relref "/doc/project/checklist.md#assert-known-added" >}}
-[LINT_SETUP]:         {{<relref "/doc/project/checklist.md#lint-setup" >}}
+[SPEC_COMPLETE]:      {{<relref "/doc/project/checklist.md#spec_complete" >}}
+[CSR_DEFINED]:        {{<relref "/doc/project/checklist.md#csr_defined" >}}
+[CLKRST_CONNECTED]:   {{<relref "/doc/project/checklist.md#clkrst_connected" >}}
+[IP_TOP]:             {{<relref "/doc/project/checklist.md#ip_top" >}}
+[IP_INSTANTIABLE]:    {{<relref "/doc/project/checklist.md#ip_instantiable" >}}
+[MEM_INSTANCED_80]:   {{<relref "/doc/project/checklist.md#mem_instanced_80" >}}
+[FUNC_IMPLEMENTED]:   {{<relref "/doc/project/checklist.md#func_implemented" >}}
+[ASSERT_KNOWN_ADDED]: {{<relref "/doc/project/checklist.md#assert_known_added" >}}
+[LINT_SETUP]:         {{<relref "/doc/project/checklist.md#lint_setup" >}}
 [D1_REVIEWED]:        {{<relref "/doc/project/checklist.md#d1-reviewed" >}}
```

Comment:
any reason why the d1-reviewed (and d2- and d3-) are still using dash vs underscore?

Reply:
Ah these have been removed in all other files and the template, except for these two instances. That's why they still used the old format. I've deleted them from this checklist as well.